domains/asheer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "itsasheer",
+        "email": "@itsasheer on Discord"
```

Comment:
You need to put your email in the `email` field, for discord you have to add a `discord` field
if you don't want to share your email you can also keep it as an empty string (`""`)